### DISCUSSION TITLE
Boot installation stuck at "Starting automated install........"

### DISCUSSION THREAD

**Discussion by imranmanraj** [Security-Onion-Solutions/securityonion#12363] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

48

### RAM

256

### Storage for /

1 TB

### Storage for /nsm

1 TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have a brand new physical server and I'm attempting to boot to the Security Onion 2.4 ISO. Running into the following issue as the dots continue.
![SO_boot_issue](https://github.com/Security-Onion-Solutions/securityonion/assets/160175302/57312b51-0c06-456c-93b5-0a6ec7d4251d)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
Did you verify the ISO download ?
Is the ISO virtually mounted to the server using something like idrac or did you flash a USB with the ISO?

**Reply by imranmanraj** [reply]
@reyesj2 yes I did verify the ISO, it is virtually mounted using iDrac

**Reply by reyesj2** [reply]
Is the server on the same network as you or are you connected over a VPN? 

If you boot your server and get to the same screen as the image you shared earlier. 
Press SHIFT + TAB and then when in the shell look in ```/var/log/anaconda/``` and share the contents of the files named ks-script* 
If the path /var/log/anaconda is wrong you should be able to find the files if they exist by running
```find / -name ks-script*```
